### PR TITLE
Remake the package for WAFS removal from GFS

### PR DESCRIPTION
**Pull request by KateFriedman-NOAA**
# Description

This PR remakes the release branch for the WAFS removal from the GFS due to implementation order change. It includes changes previously made in the original release branch:

- updated release notes
- remove all WAFS scripts and settings
- include resource update for enkfgdas_update job

Refs #2591

# Type of change

Production update

### CODE REVIEW

**Review by aerorahul — APPROVED** [review]
lgtm

File: docs/Release_Notes.md [hunk 1]
```diff
@@ -1,21 +1,21 @@
-GFS V16.3.20 RELEASE NOTES
+GFS V16.3.21 RELEASE NOTES
 
 -------
 PRELUDE
 -------
 
-Annual CO2 fix file update for 2025. Additionally, the GSI is updated to turn on assimilation of PlanetiQ GPS Radio Occultation (GPS-RO) data.
+The WAFS is separated from the GFS and is now its own package in production as WAFS.v7.0.0.
```
Comment (YaliMao-NOAA) [inline docs/Release_Notes.md:7]: It's now WAFS.v7.0.1

**Review by YaliMao-NOAA — APPROVED** [review]
The PR looks good to me, while the release note may need more updates. I can make a pull request after your merging

**Comment by KateFriedman-NOAA** [comment]
@YaliMao-NOAA Sounds good...I have now merged this, please proceed with submitting a PR to the global-workflow `release/gfs.v16.3.21` branch for release notes updates. Please also include the updated WAFS version. Thanks!

**Comment by YaliMao-NOAA** [comment]
Sure. I am working on it.

**Pull request merged**